syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "cakelab/arflow_grpc/v1/device.proto";
import "cakelab/arflow_grpc/v1/session.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message LeaveSessionRequest {
  SessionUuid session_id = 1;
  Device device = 2;
}
